README.md

```diff
@@ -295,7 +299,44 @@ This works for Source and Picture elements as well.
 
 #### Background mode
 
-This feature has been removed from react-imgix when `sizes` and `srcset` was implemented. It was decided that it was too hard to implement this feature consistently. If you would still like to use this feature, please give this issue a thumbs up: [https://github.com/imgix/react-imgix/issues/160](https://github.com/imgix/react-imgix/issues/160) If we get enough requests for this, we will re-implement it.
+Images can be rendered as the background of a container around some children by using `<Background />`. The component will measure the natural size of the container as determined by the CSS on the page, and will render an optimal image for those dimensions.
```

Comment:
Maybe it could use "behind" instead? Here are some options:

- Images can be rendered as a background behind children by using `<Background />`.
- `<Background />` allows an image to be the background behind its children.

Reply:
I like the first one best.

Reply:
It is done.